src/commands.ts

```diff
@@ -15,16 +15,19 @@ var pjson = require('../package.json');
 
 const bs58 = require('bs58');
 
+function account(_this: any): Account {
```

Comment:
What is the reason for replacing `this.account` with `account(this)`? I am just curious.

Reply:
It's just a jQuery peculiarity. `this` pointer passed to the function which called directly from the terminal (e.g. `transferToken(to: string, amount: string)` or `depositShielded(amount: string, times: string)`). But in the internal subroutines (like `prinfForcedExit(_this: any, fe: any): Promise<void>`) `this` pointer becomes `undefined` and we need find a way to get `account` object. So this ugly crutch made for that :)

Reply:
Got it, thank you for the explanation.